Services/Dashboard/Administration/js/SortationUserInputHandler.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @type {(view: int) => void}
+ */
+const handleUserInputForSortationsByView = (function() {
```

Comment:
This constant polutes the global namespace, please check our JS guidelines, e.g.: https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/docs/development/js

And: Why did you chose an upper case file name here, and a lower case filename in the other new JS file?